### DISCUSSION TITLE
Change labels on refinementList widget

### DISCUSSION THREAD

**Discussion by mikebhart** [algolia/instantsearch#4503] [General]
Im using instantserach js refinementList widget.  Currently the facet has 3 values but the label gets displayed as the field name.  Can you give me the templates syntax to change the field names to given label names.

**Comment by Haroenv** [comment]
I don't understand what you mean, could you create what you have so far in a sandbox environment? We have a template available for that purpose [here](https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/instantsearch.js). Thanks!

**Comment by mikebhart** [comment]
In that example 'Insignia', 'Samsung' checkboxes, Are they the field names or label names?  i want to display a descriptive label text next to the checkbox, not the field name.

**Reply by Haroenv** [reply]
this is possible by having the description as part of the facet value: https://discourse.algolia.com/t/use-object-as-facet-filter/10841/2
